setup.py

```diff
@@ -12,7 +12,9 @@
 from setuptools import setup
 from setuptools.extension import Extension
 
-include_dirs = [os.path.dirname(get_python_inc())]
+include_dirs = [os.path.dirname(get_python_inc())] + [
```

Comment:
Is this leftover from debugging sessions ?

Reply:
Assuming you are meaning the line below. Yeah this should already be the same as `os.path.dirname(get_python_inc())` unless I'm missing something.

Reply:
Yes, this was included by mistake :)